include/swift/AST/ModuleLoader.h

```diff
@@ -154,6 +155,10 @@ class ModuleLoader {
 
   /// Verify all modules loaded by this loader.
   virtual void verifyAllModules() { }
+
+  /// Discover overlays declared alongside this file and add infomation about
+  /// them to it.
+  void findOverlayFiles(SourceLoc diagLoc, ModuleDecl *module, FileUnit *file);
```

Comment:
This seems like a good place to use `swift/Basic/Located` -- what do you think? The `diagLoc` corresponds to the import of the `ModuleDecl`, right?

Reply:
`diagLoc` is the location where the `ModuleDecl` was imported, but it's not the location of the `ModuleDecl` itself. If this were an `ImportDecl` (and that class didn't have embedded locations), or if it were an `Identifier moduleName` along with the source location of the identifier, I would agree with you.

Reply:
Gotcha', makes sense.